guide/10-mapping-and-visualization-alt/part7_utility_functions_for_the_map_widget.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>When <code>hide_mode_switch</code> is set to <code>True</code>, the 2D/3D switch button will be hidden from the widget.  </p><p> </p><p>  <strong>Note:</strong> Once the button is hidden, it cannot be made visible again. A new <code>MapView</code> instance must be created to see the button. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>